subsys/bluetooth/controller/ll_sw/ull_sync_iso.c

```diff
@@ -605,26 +598,39 @@ void ull_sync_iso_setup(struct ll_sync_iso_set *sync_iso,
 	 */
 
 	if (IS_ENABLED(CONFIG_BT_CTLR_SYNC_ISO_RESERVE_MAX)) {
-		/* Maximum time reservation for both sequential and interleaved
+		uint32_t ctrl_spacing;
```

Comment:
```suggestion
		uint32_t ctrl_spacing_us;
```

Suggest to follow the naming with suffixing the unit

Reply:
Fixed.